arch/Kconfig

```diff
@@ -24,6 +24,7 @@ config ARC
 	imply XIP
 	select ARCH_HAS_THREAD_LOCAL_STORAGE
 	select ARCH_SUPPORTS_ROM_START
+	select ARCH_HAS_DIRECTED_IPIS if SMP
```

Comment:
Nit: this feature doesn't require SMP=y, which is a higher level abstraction in the kernel, not the arch layer.  It's enough to just declare that you can do it, the kernel will then use the API to implement whatever it wants, which may or may not be an SMP kernel.  See tests/kernel/mp for examples of exercising the MP layer of the arch code without building an SMP kernel.

Reply:
I can fix that in each of the places.